repository-management.md

```diff
@@ -0,0 +1,101 @@
+
+#### Command
+
+The labels that define what exact command a given issue/pull request applies to are prefixed with `command:`
+
+_Example: `command:cli-update`_
+
+#### Scope
+
+The labels that define what scope the current issue/pull request applies to are prefixed with `scope:`.
+
+Used scopes:
+
+* `bootstrap` - Part of the bootstrap process, which loads both WP-CLI as well as WordPress Core.
+* `command` - Part of an individual command.
+* `distribution` - Part of the distribution process, where the Phar is being built and releases are produced.
+* `documentation` - Part of the handbook, command reference or inline help.
+* `framework` - Part of the WP-CLI framework itself, which provides the architecture, API and helper functions to make commands possible.
+* `testing` - Part of the unit or functional tests.
+* `website` - Part of the website infrastructure on `wp-cli.org` or `make.wordpress.org`.
+
+_Example: `scope:documentation`_
+
+#### State
+
+The labels that defined what state a given issue/pull request is in are prefixed with `state:`.
+
+Used states:
+
+* `unconfirmed` - The bug/problem in the issue could not be replicated yet or might be related to the reporter's environment.
+* `unsupported` - The issue is outside of the scope of a bug report and cannot be supported on GitHub. The reporter should be pointed towards one of the [support channels](http://wp-cli.org/#support). 
+
+_Example: `state:unconfirmed`_
+
+### Required Labels
+
+Some labels have a special meaning and/or might be used for automated workflows down the road. These are required across all official packages.
+
+#### Bug
+
+The `bug` label denotes an issue that is a confirmed bug where the code does not produce the expected result.
+
+#### Good First Issue
+
+The `good-first-issue` label denotes an issue that is a good entry point for new contributors wanting to get their feet wet.
+
+Good first issues are small in scope and don't require extensive technical expertise or historical project knowledge to get started.
+
+## Commits
+
+No direct commits are to be done against the `master` branch of the packages. All code changes need to go through a pull request workflow. 
+
+## Pull Requests
+
+All code changes go through a pull request.
+
+Every submitted pull request needs to go through a code review, and needs to be approved by at least one of the [committers](https://github.com/orgs/wp-cli/teams/committers).
+
+Non-trivial pull requests should be preceded by a related issue that defines the problem to solve and allows for discussion of the most appropriate solution before actually writing code.
+  
+When a pull request was submitted by one of the [committers](https://github.com/orgs/wp-cli/teams/committers), the submitter should set the "Reviewers" for that pull request to `wp-cli/committers` if a general code review is needed, or to one or more specific committer profiles if the expertise of a specific person is needed/wanted.
```

Comment:
@schlessera Shouldn't it be "When a pull request **is** submitted"

Reply:
The sentence refers to the one who actually did the submission. The submission itself is in the past then.

But I think `If` would be preferable to `When` in this case.

Reply:
https://github.com/wp-cli/handbook/pull/127/commits/a19d28f2a3551852272db5060852b8b43e78f745

Reply:
Yes, `If` would be preferable. Because the sentence sounds odd when it begins with `When a pull request was submitted... submitter should set the...`